sdk/eventhub/Azure.Messaging.EventHubs.Shared/src/Diagnostics/DiagnosticProperty.cs

```diff
@@ -38,9 +35,10 @@ internal static class DiagnosticProperty
 
         /// <summary>
         ///   The activity name associated with events.
+        ///   TODO (limolkova): check it does not break appinsights SDK/visualizations
         /// </summary>
         ///
-        public static string EventActivityName { get; } = "EventHubs.Message";
```

Comment:
This is part of #39166, but I don't really like this change and would rather rename SB one to include ServiceBus.

The problem is that this one now comes from we have `Azure.Messaging.EventHubs.EventHubs` source and if we rename SB one it'd come from `ServiceBus.Servicebus` 🤣

Reply:
Changed to `EventHubMessage.Create`, checked AppInsights - it does not care, naming suggestions are very welcome

Reply:
Why not use `Message` for both, if we're already fully qualifying with the root namespace?   `Azure.Messaging.EventHubs.Message` and `Azure.Messaging.ServiceBus.Message`?  

Or, should we consider using `event` to describe Event Hubs messages, which is what they are identified as in that ecosystem.

The `Create` part of the suggestion may be confusing, as we're not instrumenting the constructor.  Is that intended to mean "we sent to the service and the service created the data?"

Reply:
I thought if we didn't include the dot in the activity name, the whole activity name would be included in the activity source name per the changes in this PR. So wouldn't just using `Message` as the activity name (as Jesse suggested) give us the activity source we want?

Reply:
> So wouldn't just using Message as the activity name (as Jesse suggested) give us the activity source we want?

Absolutely, but then it'd be inconsistent with every other span we create and the guidelines where we tell to do `Client.Method`.

For the consistency perspective, I'd prefer the current naming (with potential corrections around Message/Event for EH)

Reply:
But these spans are different as there isn't really a specific client or method to assign.

Reply:
ok, will change back:) 
We'll also have displayName that follows otel semconv naming recommendations and at that point activity name will be deep technical detail.